src/verify_ce_config.py

```diff
@@ -52,30 +55,30 @@ def warn(msg):
     # Warn users if they've set_ attributes that would be overriden by eval_set in the JOB_ROUTER_DEFAULTS
     overriden_attr = EVAL_SET_DEFAULTS.intersection(set(entry.keys()))
     if overriden_attr:
-        warn("%s in JOB_ROUTER_ENTRIES will be overriden by the JOB_ROUTER_DEFAULTS." \
-             % ', '.join(overriden_attr) \
+        warn("%s in JOB_ROUTER_ENTRIES will be overriden by the JOB_ROUTER_DEFAULTS."
+             % ', '.join(overriden_attr)
              + " Use the 'eval_set_' prefix instead.")
 
     # Ensure that users don't set the job environment in the Job Router
     if is_osg and any(x.endswith('environment') for x in entry.keys()):
-        error("Do not use the Job Router to set the environment. Place variables under " + \
-              "[Local Settings] in /etc/osg/config.d/40-localsettings.ini")
+        error("Do not use the Job Router to set the environment. Place variables under "
+              + "[Local Settings] in /etc/osg/config.d/40-localsettings.ini")
 
     # Warn users about eval_set_ default attributes in the ENTRIES since their
     # evaluation may occur after the eval_set_ expressions containg them in the
     # JOB_ROUTER_DEFAULTS
     no_effect_attr = DEFAULT_ATTR.intersection(set([x for x in entry.keys() if x.startswith('eval_set_')]))
     if no_effect_attr:
-        warn("%s in JOB_ROUTER_ENTRIES " % ', '.join(no_effect_attr) + \
-             "may not have any effect. Use the 'set_' prefix instead.")
+        warn("%s in JOB_ROUTER_ENTRIES " % ', '.join(no_effect_attr)
```

Comment:
I can't do multi-line suggestions but consider doing
```
        warn("%s in JOB_ROUTER_ENTRIES may not have any effect."
             " Use the 'set_' prefix instead." % ','.join(no_effect_attr))
```
instead. Same for the 'overriden by the JOB_ROUTER_DEFAULTS' bit above.